Unicode/Makefile.am

```diff
@@ -66,6 +66,8 @@ libgunicode_la_SOURCES =	\
 	ustring.c		\
 	utype.c			\
 	gwwiconv.c		\
+	uninameslist.c		\
+	uninameslist.h		\
```

Comment:
Will probably need to fix this, somehow this header is getting excluded from the dist tarball.

Reply:
How do I see this?

Reply:
I see the Travis failure now, but I can’t even run `make dist` here, apparently no one tried it on macOS since 2013 at least!

Reply:
Ah, the header is included in the dist tarball, the build was failing because I was passing `"-I$(top_builddir)/Unicode"` instead of `"-I$(top_srcdir)/Unicode"`. I overlooked it locally because I had libuninameslist installed.